_data/shubhi-agarwal/shubhi-agarwal.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shubhi Agarwal
+institution: The LNM Institute of Information Technology
+profile_pic: v.jpg
```

Comment:
Hey @shubhi-agarwal , I guess `v.jpg` should be `v.png` in accordance with your file.

Reply:
```suggestion
profile_pic: v.png
```